app/views/restaurants/show.html.erb

```diff
@@ -47,6 +47,16 @@
       <% @restaurant.reviews.order('created_at desc').each do |review| %>
         <div>
           <div>
+            <% if current_user == review.user %>
+              <div>
+                <%= link_to "Delete review", review_path(review), method: :delete %>
+              </div>
+            <% end %>
+            <% if current_user == review.user %>
+              <div>
+                <%= link_to "Edit Review", edit_restaurant_review_path(review.restaurant, review) %>
+              </div>
+            <% end %>
```

Comment:
You are calling the same `if current_user == review.user` check twice, so you can combine the conditional into here.